common/messages/messages_en.go

```diff
@@ -170,6 +170,34 @@ var messagesEn = map[string]util.Message{
 		RC:          500,
 		Action:      "Wait for file share deletion",
 	},
+	"ListSubnetsFailed": {
+		Code:        "ListSubnetsFailed",
+		Description: "Unable to fetch list of subnet.",
+		Type:        util.RetrivalFailed,
+		RC:          500,
+		Action:      "Unable to list subnet. Target to appropriate region 'ibmcloud target -r <region>' and verify if 'ibmcloud is subnets' is returning the subnets. If it is not returning then raise ticket for VPC team else raise ticket for IKS team.",
```

Comment:
`Verify if 'ibmcloud is subnets' is returning the subnets. Make sure you are targeting the right region 'ibmcloud target -r <region>'. If the subnet list is empty please raise a ticket for VPC team else raise a ticket for IKS team.`

Reply:
Will take it up in next PR